src/cpp/session/SessionOptions.cpp

```diff
@@ -87,7 +87,7 @@ core::ProgramStatus Options::read(int argc, char * const argv[], std::ostream& o
    core::system::unsetenv(kMonitorSharedSecretEnvVar);
 
    // compute the resource path
-   Error error = core::system::installPath("..", argv[0], &resourcePath_);
+   Error error = core::system::installPath(RSTUDIO_RESOURCES_DIR, argv[0], &resourcePath_);
```

Comment:
Maybe I'm misreading this, but it seems as though this will now expand to `.` instead of the previous value of `..`. Does this work correctly when `RSTUDIO_RESOURCES_DIR` is not supplied?